pkg/backingstore/reconciler.go

```diff
@@ -1187,6 +1178,18 @@ func (r *Reconciler) needUpdate(pod *corev1.Pod) bool {
 		r.Logger.Warnf("Change in Image Pull Secrets detected: NoobaaSecret(%v) Spec(%v)", noobaaSecret, podSecrets)
 		return true
 	}
+
+	// In case noobaa CR was changed and a new toleration was added
+	// we want that change to pass to the backingstore pod as well
+	podTolerations := pod.Spec.Tolerations
+	noobaaTolerations := r.NooBaa.Spec.Tolerations
+	if len(noobaaTolerations) > 0 {
```

Comment:
I think we can remove this check. Range does this check for us, even if the value is `nil` `range` will manage it just fine.

Reply:
removed, thanks!